core/internal/paths/longestcommonprefix_test.go

```diff
@@ -0,0 +1,40 @@
+package paths_test
```

Comment:
add the silly example of using common prefix regardless of the parts?

Reply:
that's why the test uses "/one/two", "/one/two/three" and "/one/ten", for which the longest common string prefix is "/one/t"